pom.xml

```diff
@@ -183,7 +183,7 @@
         <maven.plugin.javadoc.version>3.3.0</maven.plugin.javadoc.version>
         <!-- DO NOT bump 4.4.0, see https://github.com/apache/incubator-kyuubi/pull/441 -->
         <!-- DO NOT bump 4.5.3, see https://github.com/apache/incubator-kyuubi/issues/708 -->
-        <maven.plugin.scala.version>4.3.0</maven.plugin.scala.version>
+        <maven.plugin.scala.version>4.5.6</maven.plugin.scala.version>
```

Comment:
Unnecessary change?

Reply:
I will rollback it and try again. It's to resolve some compiling errors

Reply:
I tried, but failed to compile projects. It is necessary